components/multi-platform-controller/production-downstream/kflux-ocp-p01/host-config.yaml

```diff
@@ -489,6 +491,22 @@ data:
   dynamic.linux-ppc64le.max-instances: "30"
   dynamic.linux-ppc64le.allocation-timeout: "1800"
 
+  # Same as linux-ppc64le but with 200GB disk instead of default 100GB
+  dynamic.linux-d160-ppc64le.type: ibmp
+  dynamic.linux-d160-ppc64le.ssh-secret: "internal-prod-ibm-ssh-key"
+  dynamic.linux-d160-ppc64le.secret: "internal-prod-ibm-api-key"
+  dynamic.linux-d160-ppc64le.key: "prod-konflux-infra"
+  dynamic.linux-d160-ppc64le.image: "ppc64le-rhel-94-base-27nov2024"
+  dynamic.linux-d160-ppc64le.crn: "crn:v1:bluemix:public:power-iaas:dal12:a/eebc7153d0c844f9915fb6f3b9850a3b:d6faf373-c0b6-4ba9-b0bc-4f7e0a74f9d5::"
+  dynamic.linux-d160-ppc64le.url: "https://us-south.power-iaas.cloud.ibm.com"
+  dynamic.linux-d160-ppc64le.network: "4d164943-b043-4c6e-bba5-d233abf66c6a"
+  dynamic.linux-d160-ppc64le.system: "e980"
+  dynamic.linux-d160-ppc64le.cores: "2"
+  dynamic.linux-d160-ppc64le.memory: "8"
+  dynamic.linux-d160-ppc64le.max-instances: "30"
+  dynamic.linux-d160-ppc64le.allocation-timeout: "1800"
+  dynamic.linux-d160-ppc64le.disk: "200"
```

Comment:
for aws, we gave the d160 name as disk are 160, should we name these new instances d200?

Reply:
Thanks for the catch, updated!